### DISCUSSION TITLE
Fly DS125

### DISCUSSION THREAD

**Discussion by dimon495** [mahomaps/mm-v1#26] [Tested devices]
Не работают сетевые функции, постоянно выскакивают запросы на HTTP подключение
![photo_2023-05-25_23-31-44](https://github.com/mahomaps/mm-v1/assets/76395065/8bacde4f-c802-44d4-aaa3-cdebbc7631a7)
![photo_2023-05-25_23-31-10](https://github.com/mahomaps/mm-v1/assets/76395065/c14d351c-992f-49b6-b49f-c50561168811)
![photo_2023-05-25_23-31-10 (2)](https://github.com/mahomaps/mm-v1/assets/76395065/91499ee1-ab0f-43e6-a309-d4233c72c917)
![photo_2023-05-25_23-31-10 (3)](https://github.com/mahomaps/mm-v1/assets/76395065/675c53d9-baa3-4232-9ad0-a8f34fcf0395)

**Comment by Feodor0090** [comment]
Попробуй запросить обновление токена и покажи ошибку (ошибки в 1.0.0 "выписывает" только обнова токена)
